### PR TITLE
fix: use `vite@5`

### PR DESCRIPTION
**Pull request by manuel3108**
Relates #341, https://github.com/sveltejs/cli/pull/311#issuecomment-2531023527

From https://github.com/vitest-dev/vitest/issues/7014#issuecomment-2527453004
> Vitest 2 doesn't support Vite 6 and will not support Vite 6 officially. You can use resolutions or pnpm.overrides to change the version range locally, Vite 6 API is the same and should work for Vitest 2.
> 
> If you migrated to Vite 6, use Vitest 3 (currently in beta), release is planned for January.
> 
> If you see any type errors in your config, check the lockfile to confirm that you have several Vite versions installed and resolve it with your package manager.

### CODE REVIEW

File: packages/create/templates/demo/package.json [hunk 1]
```diff
@@ -13,9 +13,9 @@
 		"@neoconfetti/svelte": "^2.0.0",
 		"@sveltejs/adapter-auto": "workspace:*",
 		"@sveltejs/kit": "workspace:*",
-		"@sveltejs/vite-plugin-svelte": "^5.0.0",
+		"@sveltejs/vite-plugin-svelte": "^4.0.0",
 		"svelte": "^5.0.0",
 		"typescript": "^5.3.3",
-		"vite": "^6.0.0"
+		"vite": "^5.3.2"
```
Comment (AdrianGonz97) [inline packages/create/templates/demo/package.json:19]:
```suggestion
		"vite": "^5.4.11"
```

File: packages/create/templates/demo/package.template.json [hunk 2]
```diff
@@ -12,9 +12,9 @@
 		"@fontsource/fira-mono": "^5.0.0",
 		"@neoconfetti/svelte": "^2.0.0",
 		"@sveltejs/adapter-auto": "^3.0.0",
-		"@sveltejs/kit": "^2.9.0",
-		"@sveltejs/vite-plugin-svelte": "^5.0.0",
+		"@sveltejs/kit": "^2.0.0",
+		"@sveltejs/vite-plugin-svelte": "^4.0.0",
 		"svelte": "^5.0.0",
-		"vite": "^6.0.0"
+		"vite": "^5.0.3"
```
Comment (AdrianGonz97) [inline packages/create/templates/demo/package.template.json:18]:
```suggestion
		"vite": "^5.4.11"
```

File: packages/create/templates/library/package.template.json [hunk 3]
```diff
@@ -24,12 +24,12 @@
...
-		"@sveltejs/kit": "^2.9.0",
+		"@sveltejs/kit": "^2.0.0",
 		"@sveltejs/package": "^2.0.0",
-		"@sveltejs/vite-plugin-svelte": "^5.0.0",
+		"@sveltejs/vite-plugin-svelte": "^4.0.0",
 		"publint": "^0.2.0",
 		"svelte": "^5.0.0",
 		"typescript": "^5.3.2",
-		"vite": "^6.0.0"
+		"vite": "^5.0.11"
```
Comment (AdrianGonz97) [inline packages/create/templates/library/package.template.json:33]:
```suggestion
		"vite": "^5.4.11"
```

File: packages/create/templates/minimal/package.template.json [hunk 4]
```diff
@@ -10,9 +10,9 @@
 	},
 	"devDependencies": {
 		"@sveltejs/adapter-auto": "^3.0.0",
-		"@sveltejs/kit": "^2.9.0",
-		"@sveltejs/vite-plugin-svelte": "^5.0.0",
+		"@sveltejs/kit": "^2.0.0",
+		"@sveltejs/vite-plugin-svelte": "^4.0.0",
 		"svelte": "^5.0.0",
-		"vite": "^6.0.0"
+		"vite": "^5.0.3"
```
Comment (AdrianGonz97) [inline packages/create/templates/minimal/package.template.json:16]:
```suggestion
		"vite": "^5.4.11"
```

**Review by AdrianGonz97 — CHANGES_REQUESTED** [review]

**Comment by dominikg** [comment]
whats missing to get this merged?

**Comment by Rich-Harris** [comment]
preview: https://svelte-dev-git-preview-cli-353-svelte.vercel.app/

_this is an automated message_

**Comment by hyunbinseo** [comment]
Vitest 3.0 with Vite 6 support has been released.

- https://vitest.dev/blog/vitest-3
- https://github.com/vitest-dev/vitest/releases/tag/v3.0.0

**Pull request merged**